metadata-ingestion/src/datahub/ingestion/api/source.py

```diff
@@ -193,17 +193,15 @@ def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
                 self.ctx.pipeline_config.flags.generate_browse_path_v2_dry_run
             )
 
-        auto_lowercase_urns: Optional[MetadataWorkUnitProcessor] = None
+        auto_lowercase_dataset_urns: Optional[MetadataWorkUnitProcessor] = None
         if (
             self.ctx.pipeline_config
-            and self.ctx.pipeline_config.flags.auto_lowercase_urns
+            and self.ctx.pipeline_config.source.config
+            and self.ctx.pipeline_config.source.config.convert_urns_to_lowercase
```

Comment:
probably need a hasattr check here right? since not all source configs will actually have this field